src/main/java/net/sf/jabref/gui/undo/UndoableChangeType.java

```diff
@@ -35,12 +36,19 @@ public UndoableChangeType(BibEntry entry, String oldType, String newType) {
 
     @Override
     public String getUndoPresentationName() {
-        return "Undo: change type";
+        return Localization.lang("Undo") + ": " + getPresentationName();
```

Comment:
It looks to me that a lot of events now have the same `getUndo/RedoPresentationName`. Can this be pushed to the parent class AbstractUndoableEdit ?